endoreg_db/services/video_import.py

```
@@ -192,7 +191,197 @@ def import_and_anonymize(
+    except Exception as e:
+        logger.warning(f"Failed to signal completion status: {e}")
+        # Don't fail the entire import for this - processing was successful
+    
+    # Step 8: Refresh from database and return
+    with transaction.atomic():
+        video_file_obj.refresh_from_db()
+    
+    logger.info(f"Import and anonymization completed for VideoFile UUID: {video_file_obj.uuid}")
+    return video_file_obj
+
+def anonymize(video_file_obj: "VideoFile", processor_name: str, just_anonymization=True, method="masking", mask_config={}) -> "VideoFile":
```

Pylint (3.3.7)

[refactor] Too many local variables (25/15)

(R0914)